lib/src/variants/variant_attribute.dart

```diff
@@ -9,14 +9,15 @@ import 'variant.dart';
 class VariantAttribute<T extends Variant> extends Attribute
     with MergeableMixin<VariantAttribute<T>>, EquatableMixin {
   const VariantAttribute(
-    this.variant,
-    Mix mix,
-  ) : _mix = mix;
```

Comment:
This is changing the API signature from positional to named parameters. Since you area already changing the signature there is no need to deprecate it, you can just remove it, this is an internal class so it should be fine.